collect_app/src/main/res/drawable/baseline_check_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#000000"
+    android:viewportHeight="24" android:viewportWidth="24"
+    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="@android:color/white" android:pathData="M9,16.17L4.83,12l-1.42,1.41L9,19 21,7l-1.41,-1.41z"/>
```

Comment:
If the tint color is black it doesn't make sense to use white here right it's confusing. It should be black in both places.

Reply:
This is actually something I realise I'm really unclear on. Given we might have an icon appear in multiple places where it needs different tints, should we have multiple vector assets with different tints or just have one that's a default black (or probably `?colorOnSurface)? It seems from this and the other comments you'd lean towards the former?

Reply:
I think if it's used in only one place like here we can directly set the only color that's used in the XML file and don't think about it but generally yeah I think the first option is what I prefer.

Reply:
Ah I think I always need to set it programmatically anyway as otherwise I can't have a default `tint` in `pill.xml` as it'll always override the one in the drawable. For now, I'll keep the `setIconTint` calls and make the icons use `?colorOnSurface` as their default (which makes more sense than hardcoding black I think). Let me know if that doesn't sound right to you!